invenio_assets/assets/build/webpack.config.js

```diff
@@ -213,6 +214,14 @@ var webpackConfig = {
       dangerouslyAllowCleanPatternsOutsideProject: true,
       cleanStaleWebpackAssets: process.env.NODE_ENV === "production",   // keep stale assets in dev because of OS issues
     }),
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, "../node_modules/tinymce/skins"),
```

Comment:
minor: I would limit it to the specific files we need, maybe not the whole folder